guides/micronaut-cloud-database-azure/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "base": "micronaut-cloud-database-base",
+  "title": "Connect a Micronaut Data JDBC application to Azure Database for MySQL",
```

Comment:
The phrase "Azure Database *for* MySQL" sounds strange to me. 
What about "Connect a Micronaut Data JDBC application to an Azure MySQL Database"

Reply:
it may sound weird but it is what Azure calls it https://azure.microsoft.com/en-us/services/mysql/#overview